tests/v4l2decode.cpp

```diff
@@ -1031,5 +1031,6 @@ int main(int argc, char** argv)
 #endif
 
     fprintf(stdout, "decode done\n");
+    return 0;
```

Comment:
We can't, this will return -1, status code is a communication between application and shell

```
    input = DecodeInput::create(params.inputFile);
    if (input==NULL) {
        ERROR("fail to init input stream\n");
        return -1;
    }
```